### PR TITLE
Add security-analysis monitoring

### PR DESCRIPTION
**Pull request by klesaulnier**

### CODE REVIEW

**Review by TheMaskedTurtle — APPROVED** [review]

**Review by TheMaskedTurtle — COMMENTED** [review]
You should also add the configuration that adds prometheus endpoints in k8s/resources/study/config/... otherwise it won't work on docker and on k8s

File: docker-compose/technical/grafana/dashboards/security_analysis_metrics.json [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+    "list": []
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Security Analysis",
+  "uid": "nFhTFovIk",
```
Comment (flomillot) [inline docker-compose/technical/grafana/dashboards/security_analysis_metrics.json:640]: How do we choose this ? Can't we just give the name of the service instead ? Like `security_analysis`

File: docker-compose/technical/grafana/dashboards/security_analysis_metrics.json [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+  "version": 5
```
Comment (flomillot) [inline docker-compose/technical/grafana/dashboards/security_analysis_metrics.json:641]: I think it's version 3 on loadflow, can we have the same everywhere ?

File: docker-compose/technical/grafana/dashboards/computations_metrics.json [hunk 3]
```diff
@@ -689,8 +822,10 @@
         "current": {
           "selected": false,
           "text": "Prometheus",
-          "value": "PBFA97CFB590B2093"
+          "value": "Prometheus"
```
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/computations_metrics.json:825]: @TheMaskedTurtle is it ok to change this value?

**Pull request merged**